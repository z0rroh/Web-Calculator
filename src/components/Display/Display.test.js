import React from 'react';
import {Â shallow } from 'enzyme'
import Display from './Display'

describe('<Display/>', () => {
   let wrapp
   beforeEach(() => wrapp = shallow(<Display displayValue={''}/>))

   it('Should render correctly', () => {
      expect(wrapp).toMatchSnapshot()
   })

   it('Should render a div', () => {
      expect(wrapp.find('div').length).toEqual(1)
   })

   it('renders the value of displayValue', () => {
      wrapp.setProps({ displayValue: 'test'})
      expect(wrapp.text()).toEqual('test')
   })

});