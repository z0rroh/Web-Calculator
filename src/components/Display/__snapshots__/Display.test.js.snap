// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display/> Should render correctly 1`] = `
<div
  className="display-container"
>
  <p
    className="display-value"
  />
</div>
`;
